helm/oauth2-proxy/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: oauth2-proxy
-version: 5.0.6
+version: 5.0.7
```

Comment:
Could you rebase and bump a minor version of the chart (according to semver, this is new feature) instead of patch version?